ui/src/app/collection/detail/collection-public/collection-public.component.ts

```diff
@@ -26,7 +27,7 @@ export class CollectionPublicComponent extends Whitelabelled implements OnInit {
   results: PaginatedSkills | undefined
 
   from = 0
-  size = 50
+  size = this.pageSizeOptions?.length > 0 ? this.pageSizeOptions[0] : 50
```

Comment:
Can pageSizeOptions be a constant? We then don't have to null check it.

Reply:
Do you mean something like this?
```
  pageSizeOptions = [50, 100, 150]
  from = 0
  size = 50
```

Or maybe a constant from `size-pagination.component.ts
`

`values: number[] = [50, 100, 150]`

Reply:
const pageSizeOptions = [50, 100, 150]
...
  // initialize size, no null check, and reassign it in the handler 
  size = this.pageSizeOptions[0]